packages/typegpu/src/shared/env.ts

```diff
@@ -7,3 +7,6 @@
  */
 // biome-ignore lint/suspicious/noExplicitAny: the types are not important here
 export const DEV = (globalThis as any).process.env.NODE_ENV === 'development';
```

Comment:
```suggestion
export const DEV = globalThis.process.env.NODE_ENV === 'development';
```
There is no intellisense for `NODE_ENV`, but the types seem to be fine if I do this